Review Italian version (#50)
italia/publiccode.yml: docs/en/schema.it.rst

CAD <https://docs.italia.it/italia/piano-triennale-ict/codice-amministrazione-digitale-docs/it/v2017-12-13/_rst/capo8_art73.html>`__ (Italian language).

Key ``conforme/misureMinimeSicurezza``
''''''''''''''''''''''''''''''''''''''

- Type: boolean
- Presence: optional

If present and set to ``yes``, the software is compliant with the `Misure
minime di sicurezza ICT per le Pubbliche
amministrazioni <http://www.agid.gov.it/sites/default/files/documentazione/misure_minime_di_sicurezza_v.1.0.pdf>`__ (Italian language). 

Key ``conforme/gdpr``
'''''''''''''''''''''

- Type: boolean
- Presence: optional

If present and set to ``yes``, the software respects the GDPR.

Section ``piattaforme``
~~~~~~~~~~~~~~~~~~~~~~~

Key ``spid``
''''''''''''

- Type: boolean
- Presence: optional

If present and set to ``yes``, the software interfaces with `SPID
- il Sistema Pubblico di Identità
Digitale <https://developers.italia.it/it/spid>`__.

Key ``cie``
'''''''''''

- Type: boolean
- Presence: optional

If present and set to ``yes``, the software interfaces with the Italian
electronic ID card (``Carta di Identità Elettronica``).

Key ``anpr``
''''''''''''

- Type: boolean
- Presence: optional

> If present and set to ``yest``, the software interfaces with ANPR.

Reviewer: ```suggestion
If present and set to ``yes``, the software interfaces with ANPR.
```

Key ``pagopa``
''''''''''''

- Type: boolean
- Presence: optional

> If present and set to ``yest``, the software interfaces with pagoPA.

Reviewer: ```suggestion
If present and set to ``yes``, the software interfaces with pagoPA.
```

Section ``riuso``
~~~~~~~~~~~~~~~~~

This section contains a set of keys related to the publication of the software

> inside the `Catalogo del Riuso <https://developers.italia.it>`__.

Reviewer: ```suggestion
inside the reuse catalog of `Developers Italia <https://developers.italia.it>`__.
```
